package.json

```diff
@@ -28,12 +28,14 @@
     "prepublishOnly": "pnpm build"
   },
   "dependencies": {
-    "fflate": "^0.7.4"
+    "fflate": "^0.8.2"
   },
   "devDependencies": {
-    "prettier": "^2.7.1",
-    "tsup": "^6.2.3",
-    "typescript": "^4.8.4"
+    "@swc/core": "^1.4.8",
```

Comment:
Why do we need `@swc/core` then?

Reply:
![image](https://github.com/reslear/dotlottie-player-core/assets/77834863/3a92d53f-c183-474f-b9cd-bd72787efe19)

We don't necessarily need it, but it was a warning by `tsup` so I thought might as well include it.

Reply:
ok, i delete emitDecoratorMetadata